docs/user/search.rst

```diff
@@ -44,6 +44,9 @@ Fields
    String position in the translation file.
 ``added:DATETIME``
    Timestamp for when the string was added to Weblate.
+``added:TEXT``
+   String containing adverb of time like ``yesterday``, ``last month``, and ``2 days ago``.
+   It also supports :ref:`search-operators`.
```

Comment:
Instead of repeating it in every field, I'd add below a chapter on the topic and reference it from individual fields.